services/rollout-service/pkg/argo/argo.go

```diff
@@ -238,6 +238,8 @@ func CreateArgoApplication(overview *api.GetOverviewResponse, app *api.Environme
 	annotations["argocd.argoproj.io/manifest-generate-paths"] = "/" + manifestPath
 	labels["com.freiheit.kuberpult/team"] = team(overview, app.Name)
 
+	var syncWindows v1alpha1.SyncWindows
+
```

Comment:
nitpick: why move it up here?

Reply:
Argh, this is leftovers of something I tried, this can be reverted